gradio/components.py

```diff
@@ -943,8 +962,18 @@ def preprocess(self, x: List[str]) -> List[str] | List[int]:
         Parameters:
             x: list of selected choices
         Returns:
-            list of selected choices as strings or indices within choice list
+            ValuError if x is None or one of the choice in x is not in the choices list
+            otherwise list of selected choices as strings or indices within choice list
         """
+
+        if x is None:
+            raise ValueError("Check box value(s) cannot be None")
```

Comment:
Remove these two lines
```suggestion
        if x is None:
            raise ValueError("Check box value(s) cannot be None")
```